core/src/main/java/software/amazon/awssdk/core/http/AmazonSyncHttpClient.java

```diff
@@ -55,15 +55,15 @@
 
 @ThreadSafe
 @SdkInternalApi
-public class AmazonHttpClient implements SdkAutoCloseable {
```

Comment:
Can you add a ReviewBeforeRelease annotation to come up with a better name for this and the async variant?